yt_dlp/extractor/vimeo.py

```diff
@@ -253,6 +253,23 @@ def _extract_original_format(self, url, video_id, unlisted_hash=None):
                             'quality': 1,
                         }
 
+    def _extract_original_format_2(self, video_id):
```

Comment:
Name could be better tbh
So open to suggestions.

Reply:
why have seperate functions if you cant even think of seperate names for them